test/integration/integration.cc

```diff
@@ -855,7 +855,7 @@ void BaseIntegrationTest::testMissingDelimiter() {
 
 void BaseIntegrationTest::testInvalidCharacterInFirstline() {
   std::string response;
-  sendRawHttpAndWaitForResponse("GET /\t HTTP/1.1\r\nHost: host\r\n\r\n", &response);
+  sendRawHttpAndWaitForResponse("GET \t HTTP/1.1\r\nHost: host\r\n\r\n", &response);
```

Comment:
I think it's possible in some versions of the HTTP library the \t is treated as whitespace.
This fix likely is failing due to missing host rather than invalid character so changing what's tested.   I'll take a look